OsLogin/synth.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+common = gcp.CommonTemplates()
+
+library = gapic.php_library(
+    service='oslogin',
+    version='v1beta',
+    artman_output_name='google-cloud-os-login-v1beta')
+
+# copy all src except partial veneer classes
+s.move(library / f'src/V1beta/Gapic')
+s.move(library / f'src/V1beta/resources')
```

Comment:
I think for OS Login, we can copy the veneer class too.
BTW, I'm also adding synthtool script in https://github.com/GoogleCloudPlatform/google-cloud-php/pull/1127

Reply:
Ok, I'll remove this script.